src/settings_type.h

```diff
@@ -532,6 +532,7 @@ struct EconomySettings {
 	uint16 town_noise_population[4];         ///< population to base decision on noise evaluation (@see town_council_tolerance)
 	bool   allow_town_level_crossings;       ///< towns are allowed to build level crossings
 	bool   infrastructure_maintenance;       ///< enable monthly maintenance fee for owner infrastructure
+	uint16 cargo_scale;                      ///< scale cargo production of towns and industries by this percentage.
```

Comment:
I never like variables that don't tell me their unit, especially for something like `scale`. Can I suggest we make it `cargo_scale_percentage` or something? Other ideas are welcome .. but `scale` suggests to me 0..1 :D